src/main/java/com/mraof/minestuck/effects/MSEffects.java

```diff
@@ -11,7 +11,7 @@
  */
 public class MSEffects
 {
-	public static final DeferredRegister<MobEffect> REGISTER = DeferredRegister.create(ForgeRegistries.MOB_EFFECTS, Minestuck.MOD_ID);
+	public static final DeferredRegister<MobEffect> EFFECTS = DeferredRegister.create(ForgeRegistries.MOB_EFFECTS, Minestuck.MOD_ID);
```

Comment:
Why was this rename made?

Reply:
I know its not directly applicable to the PR but it seemed close enough related to the PR(since the Items register was named to ITEMS) that it would be fine to do so without created a PR just for the task

Reply:
I suppose I should've phrased it about the wider naming scheme instead of a specific naming instance.

At this time, I personally think that deferred register fields should all be called `REGISTER`. (As long as it is in situations such as these where only one specific deferred register is used within one appropriately named class)

Reply:
Will make the change then! Fine going with that approach. I saw them get named individually in javadoc for a relevant class so I took that approach

Reply:
It does make sense to name them like that in the javadoc to distinguish between different deferred registers in the same class, or if the registry name is not already present in the class name.
So I think that naming scheme is more relevant for a small mods without dedicated classes for each registry.